src/lib/crypto/kimchi_backend/gadgets/foreign_field.mli

```diff
@@ -221,6 +322,7 @@ val sub :
 val mul :
      (module Snark_intf.Run with type field = 'f)
   -> 'f External_checks.t (* external_checks *)
+  -> ?bound_check_result:bool
```

Comment:
I wonder if we want to expose this quite internal flag to the user of a normal ffmul.